aggregator/flush.go

```diff
@@ -70,6 +70,13 @@ type flushRequest struct {
 	BufferAfterCutoff time.Duration
 }
 
+type eagerForwardingMode int
+
+const (
+	allowEagerForwarding eagerForwardingMode = iota
```

Comment:
nit: `eagerForwardingEnabled` and `eagerForwardingDisabled`?

Reply:
Hm that conflicts with the options `EagerForwardingEnabled` and `EagerForwardingDisabled`. This is to convey the meaning that based on the leader / follower role, a flush may be allowed or disallowed to consume metrics even when eager forwarded is enabled.